x-pack/legacy/plugins/ml/public/data_frame/pages/data_frame_new_pivot/components/step_define/step_define_form.tsx

```diff
@@ -441,19 +458,17 @@ export const StepDefineForm: SFC<Props> = React.memo(({ overrides = {}, onChange
                     defaultMessage: 'Use a query string to filter the source data (optional).',
```

Comment:
I think we should drop `string` from this label seeing as it is now using KQL, to leave `Use a query to filter the source data (optional).`